src/Framework/Microsoft.Build.Framework.csproj

```diff
@@ -1,14 +1,27 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>$(LibraryTargetFrameworks)</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
📝  I need to add this to the documentation steps in https://github.com/dotnet/roslyn/issues/49991

Reply:
Stupid question: If the code always requires this prop, can it be specified in the package (in Microsoft.CodeAnalysis.Collections.targets?)